pkg/shell/paths.go

```diff
@@ -28,8 +27,8 @@ func MakePaths(stderr io.Writer, overrides Paths) Paths {
 	p := overrides
 	setDir(&p.RunDir, "secure run directory", getSecureRunDir, stderr)
 	if p.RunDir != "" {
+		setChild(&p.RunDir, p.RunDir, "")
```

Comment:
Hmm?

Reply:
Did you miss this comment? This line of code doesn't do anything useful and is confusing, although it doesn't seem to do any harm so I can fix this after merging this PR.

Reply:
Yes, that particular change is wrong. It's left over from earlier changes to this code when I was figuring out how to adapt the code to use `ClaimFile`.